ts/screens/profile/SecurityScreen.tsx

```diff
@@ -0,0 +1,172 @@
+import React, { FC, useEffect, useState } from "react";
+import { StyleSheet } from "react-native";
+import { List, Content, View } from "native-base";
+import { connect } from "react-redux";
+import { Dispatch } from "redux";
+import TouchID, { AuthenticationError } from "react-native-touch-id";
+import I18n from "../../i18n";
+import { GlobalState } from "../../store/reducers/types";
+import { getFingerprintSettings } from "../../sagas/startup/checkAcknowledgedFingerprintSaga";
+import TopScreenComponent from "../../components/screens/TopScreenComponent";
+import { ContextualHelpPropsMarkdown } from "../../components/screens/BaseScreenComponent";
+import ListItemComponent from "../../components/screens/ListItemComponent";
+import { H1 } from "../../components/core/typography/H1";
+import { H4 } from "../../components/core/typography/H4";
+import { updatePin } from "../../store/actions/pinset";
+import { identificationRequest } from "../../store/actions/identification";
+import { shufflePinPadOnPayment } from "../../config";
+import { authenticateConfig } from "../../utils/biometric";
+import { showToast } from "../../utils/showToast";
+import { preferenceFingerprintIsEnabledSaveSuccess } from "../../store/actions/persistedPreferences";
+import customVariables from "../../theme/variables";
+
+const styles = StyleSheet.create({
+  container: {
+    paddingHorizontal: customVariables.contentPadding
+  }
+});
+
+const contextualHelpMarkdown: ContextualHelpPropsMarkdown = {
+  title: "profile.preferences.contextualHelpTitle",
+  body: "profile.preferences.contextualHelpContent"
+};
+
+type Props = ReturnType<typeof mapDispatchToProps> &
+  ReturnType<typeof mapStateToProps>;
+
+const SecurityScreen: FC<Props> = ({
+  isFingerprintEnabled,
+  requestIdentificationAndResetPin,
+  setFingerprintPreference
+}): React.ReactElement => {
+  const [isFingerprintAvailable, setIsFingerprintAvailable] = useState(false);
+
+  useEffect(() => {
+    getFingerprintSettings().then(
+      biometryTypeOrUnsupportedReason => {
+        setIsFingerprintAvailable(
+          biometryTypeOrUnsupportedReason !== "UNAVAILABLE" &&
+            biometryTypeOrUnsupportedReason !== "NOT_ENROLLED"
+        );
+      },
+      _ => undefined
+    );
+  }, []);
+
+  // FIXME: on IOS add this function onPress ListItemComponent if user refused
```

Comment:
Please, also here: explain in the FIXME comment what is the task but remove dead code